assets/js/components/adminbar/__snapshots__/AdminBarWidgets.test.js.snap

```diff
@@ -98,80 +98,24 @@ exports[`AdminBarWidgets should render the Admin Bar Widgets, including the Acti
       class="mdc-layout-grid__cell mdc-layout-grid__cell--span-3-desktop mdc-layout-grid__cell--span-2-tablet"
     >
       <div
-        class="googlesitekit-data-block overview-total-impressions googlesitekit-data-block--default"
-        tabindex="-1"
+        class="googlesitekit-preview-block"
+        style="width: auto; height: 59px;"
```

Comment:
This snapshot test is failing https://github.com/google/site-kit-wp/actions/runs/7197381611/job/19604375682?pr=7969#step:7:465

Reply:
Oh, weird, I ran the update. I'll do it again 😅